app/src/main/java/com/amaze/filemanager/asynchronous/services/ExtractService.java

```diff
@@ -326,6 +327,14 @@ public boolean isCancelled() {
             passwordProtected = true;
             paused = true;
             publishProgress(e);
+          } else if (e.getCause() != null
+              && UnsupportedRarV5Exception.class.isAssignableFrom(e.getCause().getClass())) {
+            Log.e(TAG, "RAR " + compressedPath + " is unsupported V5 archive", e);
+            AppConfig.toast(
+                extractService,
+                extractService.getString(R.string.error_unsupported_v5_rar, compressedPath));
+            paused = false;
```

Comment:
I don't see any use of `paused` variable throughout the file.
Can you check, and maybe remove if it's not required?

Reply:
Just to be sure thing. Yes, can be omitted.

Reply:
Can you check I think `paused` variable itself can be removed from class, it doesn't have any significant usage 🤔

Reply:
To pause the service while waiting for archive password, in the case of password-protected archives. See `onProgressUpdate()`